tests/Microsoft.Fx.Portability.MetadataReader.Tests/ManagedMetadataReaderTests.cs

```diff
@@ -227,6 +220,23 @@ public void MultidimensionalPrimitiveArray()
             Assert.Equal(systemObject.DefinedInAssemblyIdentity, definedInAssemblyIdentity);
         }
 
+        [Fact]
+        public void ThrowsSystemObjectNotFoundException()
+        {
+            var dependencyFilter = Substitute.For<IDependencyFilter>();
+            dependencyFilter.IsFrameworkAssembly(Arg.Any<AssemblyReferenceInformation>()).Returns(false);
+            var dependencyFinder = new ReflectionMetadataDependencyFinder(dependencyFilter);
+            var assemblyToTest = TestAssembly.Create("FilterApis.cs", false, new[] { typeof(Image).GetTypeInfo().Assembly.Location });
+            var progressReporter = Substitute.For<IProgressReporter>();
+
+            var exception = Assert.Throws<AggregateException>(() =>
```

Comment:
Why is this throwing an aggregate exception? not a blocker, but we should only be throwing a single exception

Reply:
This is wrapped inside a Parallel.ForEach for because we try to compute multiple assemblies in paralle.